contracts/CandidateHub.template

```diff
@@ -155,7 +150,11 @@ contract CandidateHub is ICandidateHub, System, IParamSubscriber {
           b. transfer the candidate's margin eth value to the SystemReward contract
   */
   function jailValidator(address operateAddress, uint256 round, uint256 fine)
-        external override onlyValidator onlyIfCandidateAddress(operateAddress) {
+        external override onlyValidator {
+    if (operateMap[operateAddress] == 0) {
```

Comment:
apparently the expected behavior in case of non-candidate is to return rather than revert